recipes/multiqc/meta.yaml

```diff
@@ -1,16 +1,16 @@
 package:
   name: multiqc
-  version: "1.3"
+  version: "1.4"
 
 source:
-  fn: v1.3.tar.gz
-  url: https://github.com/ewels/MultiQC/archive/v1.3.tar.gz
-  md5: 67c4aae8b85ae40fcb2e98319fb629ac
+  fn: v1.4.tar.gz
+  url: https://github.com/ewels/MultiQC/archive/v1.4.tar.gz
+  md5: 83a20323d7b72b588b5e2c5e5074056b
```

Comment:
For checksums that aren't provided alongside the source, we should prefer `sha256` (http://bioconda.github.io/guidelines.html#hashes).